voice/bxml/verbs/forward.md

```diff
@@ -14,7 +14,11 @@ Forwards an unanswered incoming call to another number.
 
 ### Callbacks Received
 
-None
+| Callbacks                                 | Can reply with more BXML |
+|:------------------------------------------|:-------------------------|
+| [Disconnect](../callbacks/disconnect.md)  | No                       |
+
+**NOTE:** You will not receive any callbacks for the forwarded leg of the call (i.e. Answer). 
```

Comment:
```suggestion
**NOTE:** You will not receive any callbacks for the forwarded leg of the call (e.g., Answer). 
```
You probably already know this, but `i.e.` roughly translates as "that is"; `e.g.` as "for example"; and `viz.` as "most notably".
We probably want to use `e.g.` or perhaps `viz.` to indicate that, while other callbacks exist, they're not possible from a user perspective (in the case of BXML callbacks) and not technically supported (in the case of `answerCallback`).

Reply:
Yep; he's right about the latin. It is clear otherwise.